api/agent/call.go

```diff
@@ -91,10 +87,9 @@ func FromRequest(appName, path string, req *http.Request) CallOpt {
 		}
 
 		c.Call = &models.Call{
-			ID:      id,
-			AppName: appName,
-			Path:    route.Path,
-			Image:   route.Image,
+			ID:    id,
```

Comment:
thinking about this, it's kind of unfortunate since in our apis they all take names, the api to actually get a call will be `/1/apps/:app_name/calls/:call_id` and then the body will return an app id but not an app name. at least, it seems weird to bubble this up here but i'm not sure what we want (i'm not sure we know, either)

Reply:
we need both the app name and id here, the id for storing things by app id, and the app name so that the call api doesn't change (don't think we want to expose id in the apis? not sure, needs deliberation, but at least having both seems reasonable if we don't want to make a decision)